### PR TITLE
Feat/story groups train

### PR DESCRIPTION
**Pull request by Palashx**
This branch has commits related to intro-stories

### CODE REVIEW

File: botfront/imports/api/storyGroups/storyGroups.methods.js [hunk 1]
```diff
@@ -3,6 +3,28 @@ import { check } from 'meteor/check';
 
 import { StoryGroups } from './storyGroups.collection';
 
+export const createIntroStoryGroup = (projectId) => {
+    if (!Meteor.isServer) throw Meteor.Error(401, 'Not Authorized');
+    Meteor.call(
+        'storyGroups.insert',
+        {
+            name: 'Intro Story Group',
```
Comment (znat) [inline botfront/imports/api/storyGroups/storyGroups.methods.js]: Please rename Intro Story Group to "Intro stories"

File: botfront/imports/ui/components/nlu/models/NLUTrainButton.jsx [hunk 2]
```diff
@@ -22,12 +22,12 @@ class NLUTrainButton extends React.Component {
     }
```
Comment (znat) [inline botfront/imports/ui/components/nlu/models/NLUTrainButton.jsx]: Shouldn't that component be renamed and put out of the nlu hierarchy? Coordinate with @MatthieuJnon

Comment (Palashx) [reply botfront/imports/ui/components/nlu/models/NLUTrainButton.jsx]: Will coordinate with @MatthieuJnon

File: botfront/imports/api/storyGroups/storyGroups.methods.js (hunk 1 above)
Comment (Palashx) [reply botfront/imports/api/storyGroups/storyGroups.methods.js]: Done

File: botfront/cypress/integration/stories/stories.spec.js [hunk 3]
```diff
@@ -48,8 +49,33 @@ describe('stories', function() {
         cy.contains(initialText).should('not.exist');
         cy.get('[data-cy=delete-story]').click();
         cy.dataCy('confirm-yes').click();
+        cy.wait(100);
         cy.contains(storyGroupTwo).click();
         cy.get('[data-cy=delete-story]').click();
         cy.dataCy('confirm-yes').click();
     });
+
```
[the remark this answers is not in the document]
Comment (Palashx) [reply botfront/cypress/integration/stories/stories.spec.js]: Added consistency for data-cy

File: botfront/cypress/integration/stories/stories.spec.js [hunk 4]
```diff
@@ -48,8 +49,33 @@ describe('stories', function() {
...
+        cy.get('[data-cy=edit-name] input').click().type('{backspace}{backspace}{backspace}{enter}');
+        cy.contains('storyGroup').should('exist');
+        cy.contains('Train');
+        cy.get('.active > #not-selected').click();
+        cy.contains('Train 1 story group');
+
+        cy.contains('storyGroup').click();
+        cy.get('[data-cy=delete-story]').click();
+        cy.dataCy('confirm-yes').click();
+    });
```
[the remark this answers is not in the document]
Comment (Palashx) [reply botfront/cypress/integration/stories/stories.spec.js]: Added these tests

File: botfront/imports/ui/components/nlu/models/NLUTrainButton.jsx (hunk 2 above)
Comment (MatthieuJnon) [reply botfront/imports/ui/components/nlu/models/NLUTrainButton.jsx]: I agree it's used everywhere now, @Palashx you can move that file to a more general folder

File: botfront/imports/ui/components/common/Browser.jsx [hunk 5]
```diff
@@ -48,55 +74,98 @@ class Browser extends React.Component {
...
+                
             </Menu.Item>
         ));
-
-        if (allowAddition) {
-            items.unshift(
-                !addMode ? (
-                    <Menu.Item
+        return (
+            <React.Fragment>
```
Comment (MatthieuJnon) [inline botfront/imports/ui/components/common/Browser.jsx]: You don't have to write React.Fragment, you can just put empty brackets <> </> 
https://reactjs.org/docs/fragments.html#short-syntax

File: botfront/imports/ui/components/common/style.less [hunk 6]
```diff
@@ -0,0 +1,36 @@
+.ui.vertical.menu .item>#selected {
```
Comment (MatthieuJnon) [inline botfront/imports/ui/components/common/style.less]: try and leverage .less possibilities, for instance you could have put .ui.vertical.menu only once and then nest other declarations inside. Come and see me if you don't understand what I mean

File: botfront/imports/ui/components/stories/StoriesEditor.jsx [hunk 7]
```diff
@@ -179,18 +179,7 @@ function StoriesEditor(props) {
...
-                            name='add'
-                            link
-                            onClick={onAddNewStory}
-                            size='large'
-                            data-cy='add-story'
-                        />
-                    )}
-                    content='Add a story'
-                />
+                <Button icon='add' basic name='add' onClick={onAddNewStory} size='medium' data-cy='add-story' color='black' content='Add a story' />
```
Comment (MatthieuJnon) [inline botfront/imports/ui/components/stories/StoriesEditor.jsx]: Why did you remove the popup ?

File: botfront/imports/ui/components/nlu/models/NLUTrainButton.jsx (hunk 2 above)
Comment (Palashx) [reply botfront/imports/ui/components/nlu/models/NLUTrainButton.jsx]: cool will do it and make it a generic train button which can be later on used at any place

File: botfront/imports/ui/components/stories/StoriesEditor.jsx (hunk 7 above)
Comment (Palashx) [reply botfront/imports/ui/components/stories/StoriesEditor.jsx]: Because now the button has the same text, as the popup so it will be redundant

File: botfront/imports/ui/components/common/style.less (hunk 6 above)
Comment (Palashx) [reply botfront/imports/ui/components/common/style.less]: I will come and see you

Comment (Palashx) [reply botfront/imports/ui/components/common/style.less]: Done
